files/helm/fastgpt/templates/configmap-env.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
```

Comment:
Please use secrets to store those values.

Reply:
Sure.

![image](https://github.com/labring/FastGPT/assets/1450685/97583305-9637-4eed-a83b-ffc215af9dc7)

Reply:
See also https://github.com/labring/FastGPT/pull/1023/commits/dea1a3460e98ce4f85e49b5a12e5cc977fbd533c